source/best-practices/backup.bat

```diff
@@ -0,0 +1,22 @@
+REM
+REM  目的： 使用 robocopy 命令备份指定目录
+REM  作者： 李天觉
+REM  最近修改日期： 2021年2月4日
+REM
+
+echo Check if the source and backup paths exist
+pause

```

Comment:
Actually, I don't know what will happen if source folder, the backup folder `e:\backup` and its subfolders `e:\backup/directory1` , `e:\backup/directorys`  do not exist.

Reply:
> 1、执行脚本会自动创建备份目录及子目录；

Great~

> 2、如果 `source` 文件夹不存在会报错 `系统找不到文件`；

This is not a big problem. So I deleted those two lines.

> 3、如果文件夹的名字有中文需要把文件格式改为 `ANSI`；

A note is added in `backup.bat`. See   6a0f501.

> 4、如果文件夹的名字有空格需要加引号，比如 `robocopy "e:\我 测试 一下" e:\back\ /mir /mt /log+:e:\backup.txt`

I added `"` for all the folders and files. See  6a0f501.